graphics/timg/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github  1.0
+
+github.setup        hzeller timg 1.6.0 v
+github.tarball_from archive
+revision            0
+categories          graphics
+
+maintainers         {outoforder.cc:urkle @urkle} openmaintainer
+
+description         Terminal Image and Video Viewer
+long_description    A user-friendly terminal image viewer that uses graphic \
+                    capabilities of terminals (Sixel, Kitty or iterm2), or 24-Bit \
+                    color capabilities and unicode character blocks if these are not \
+                    available.
+
+license             GPL-2
+homepage            https://timg.sh/
+
+
+checksums           rmd160  fc1b39f4395f7a4dc4da77f12e558289e05f3425 \
+                    sha256  9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af \
+                    size    1166246
+
+installs_libs       no
+
+depends_build-append \
+                    path:bin/pkg-config:pkgconfig
+
+depends_lib-append  port:GraphicsMagick \
+                    port:webp \
+                    path:include/turbojpeg.h:libjpeg-turbo \
+                    port:libexif \
+                    path:lib/pkgconfig/librsvg-2.0.pc:librsvg \
+                    path:lib/pkgconfig/cairo.pc:cairo \
+                    port:ffmpeg \
+                    port:openslide \
+                    path:lib/pkgconfig/poppler.pc:poppler \
+                    port:pandoc
+
+configure.args-append \
+                    -DWITH_LIBSIXEL=OFF
+
```

Comment:
please also add here:
`-DTIMG_VERSION_FROM_GIT=OFF`

it's not completely clear to me whether it's actually doing something, but if as you can see from the `CMakeLists.txt` file if enabled it will look for `git` as well. That's could be an issue on older systems and you don't explicitly depend on the `git` port. The version number is perfectly fine without this option, so I'd suggest to add it.

Reply:
it's in the src/CMakeLists.txt if that is on it uses git to grab the version from tags and commit time.

When turned off, it uses the cmake defined version in the main CMakeLists.txt, however it usets the GIT_DATE variable which is part of the version.

However, since we are not building from a git checkout, it makes ZERO difference as it always uses that fallback no-git approach anyways.

So, yes disabling this would be the safest option.